cmd/yggctl/actions.go

```diff
@@ -208,14 +211,23 @@ func workersAction(c *cli.Context) error {
 		fmt.Println(string(data))
 	case "table":
 		writer := tabwriter.NewWriter(os.Stdout, 4, 4, 2, ' ', 0)
-		fmt.Fprintf(writer, "WORKER\tFEATURES\n")
+		_, err = fmt.Fprint(writer, "WORKER\tFEATURES\n")
```

Comment:
```suggestion
		_, err := fmt.Fprint(writer, "WORKER\tFEATURES\n")
```

Reply:
No, I do not want to mix fixing issue and doing refactoring of code.